docker/tests/scripts/test-fedora-binaries.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+set -euo pipefail
+
+for version in $(jq -r '.fedora[]' docker/supported_versions.json); do
+    docker build --build-arg dist="$version" --build-arg repo="Tezos" -t fedora-test -f docker/tests/Dockerfile-fedora-test .
+    docker run fedora-test
```

Comment:
```suggestion
    docker run --rm fedora-test
```
To cleanup the container after finishing

Reply:
I was thinking about it, but since these scripts are only used in CI, the container will anyway be deleted afterwards

Reply:
> the container will anyway be deleted afterwards

No, this is not true, containers will only be deleted after daily `docker system prune` invocation on CI server

Reply:
Seems to be resolved now